tikv-control.md

```diff
@@ -415,7 +419,7 @@ success!
 > **Note:**
 >
 > - This command only supports the local mode. It prints `success!` when successfully run.
-> - You must run this command for all stores where specified Regions' peers are located. If `-r` is not set, all Regions are involved, and you need to run this command for all stores.
+> - You must run this command for all stores where specified Regions' peers are located. If `-r` is not set, all Regions are involved, and you need to run this command for all stores. You need to restart the healthy stores before restoring them. Otherwise, the inconsistency of the peer lists in Region replicas will cause an error when you execute `split-region` or `remove-peer`, resulting in the inconsistency of other metadata, and finally the Regions will become unavailable.
```

Comment:
@Connor1996 There is an evident inconsistency between the Chinese and the English for this note. Please help determine which version prevails. Thanks!

- The Chinese version: "如果使用 `--all-regions`，通常需要在集群剩余所有健康的 store 上执行此命令。"
- The English version: "You must run this command for all stores where specified Regions' peers are located. If `-r` is not set, all Regions are involved, and you need to run this command for all stores."
```suggestion
> - You must run this command for all stores where specified Regions' peers are located. If `-r` is not set, all Regions are involved, and you need to run this command for all stores. You need to ensure that the healthy stores stop providing services before restoring these stores. Otherwise, the inconsistent peer lists among Region replicas will cause an error when you execute `split-region` or `remove-peer`, which causes inconsistency between other metadata, and finally, the Regions will become unavailable.
```

Reply:
The Chinese version prevails.

Reply:
@Liuxiaozhen12 Please address this comment and align the first sentence with the Chinese. Thanks!